### DISCUSSION TITLE
加入 YesPlayMusic 交流群组

### DISCUSSION THREAD

**Discussion by qier222** [qier222/YesPlayMusic#1151] [Announcements]
Telegram群组：[https://t.me/yesplaymusic](https://t.me/yesplaymusic)
Matrix群组：#1591

**Comment by Leonliang0520** [comment]
Add  Mica UI on Windows 11.

**Comment by ggad23** [comment]
Very easy to use！！I hope to add local music playback and the ability to search for lyrics online in future releases.

**Comment by ocean9999** [comment]
安卓版是哪个
